components/nav/Phone.tsx

```diff
@@ -0,0 +1,55 @@
+import { Avatar, Dropdown } from 'flowbite-react';
+import { signIn, signOut, useSession } from 'next-auth/react';
+import Image from 'next/image';
+import React, { useState } from 'react';
+
+const Phone = () => {
+  const [token, setToken] = useState(false);
+  const { data: session } = useSession();
+  console.log(session);
+
+  return (
+    <Dropdown
+      arrowIcon={false}
+      inline={true}
+      label={<Avatar alt="User settings" img={'/images/more.png'} size="xs" />}
```

Comment:
```suggestion
      label={<Avatar alt="User settings" img="/images/more.png" size="xs" />}
```
img에 중괄호는 필요없을 것 같네요!